scripts/pre_push_hook.py

```diff
@@ -330,6 +331,27 @@ def main():
             if frontend_status != 0:
                 print 'Push aborted due to failing frontend tests.'
                 sys.exit(1)
+    try:
+        with open('successful_push_counter.txt', 'r+') as f:
+            successful_push_count = f.read()
+    except IOError:
+        with open('successful_push_counter.txt', 'w') as f:
+            successful_push_count = '0'
+            f.write(successful_push_count)
+    if int(successful_push_count) < SUCCESSFUL_PUSH_THRESHOLD:
+        sys.stdin.flush()
+        sys.stdin = open('/dev/tty', 'r')
```

Comment:
Why these lines? I do not understand. What is their purpose? Also, `/dev/tty` requires sudo permissions, which is definitely not what we want.

Reply:
The `raw_input` command that I used encounters EOF on call and hence exits abruptly without taking the input. Using the above set of commands, it works properly.
And about sudo permissions... I missed this point and will think of an alternative.
Thanks.